### PR TITLE
editorial: abstract, less is more (general)

### PR DESCRIPTION
**Pull request by peppelinux**
This PR doesn't specify that the server is an Authorization Server, using less words.

### CODE REVIEW

**Comment by giadas** [comment]
I was wondering if it is better to remove “OAuth 2.0” at least in the title and in the abstract and write something more general like: 

> This document details how a client can learn about an endpoint to obtain a Nonce generated by a server (Nonce endpoint) and provides security guidelines for server-provided Nonce generation and verification.

In the introduction, we can then specify “While we provide the details to integrate this endpoint in OAuth 2.0 implementations [[RFC6749](https://peppelinux.github.io/draft-demarco-nonce-endpoint/draft-demarco-nonce-endpoint.html#RFC6749)], our approach and the identified security considerations about Nonce are general.”

**Comment by giadas** [comment]
> I was wondering if it is better to remove “OAuth 2.0” at least in the title and in the abstract and write something more general like:
> 
> > This document details how a client can learn about an endpoint to obtain a Nonce generated by a server (Nonce endpoint) and provides security guidelines for server-provided Nonce generation and verification.
> 
> In the introduction, we can then specify “While we provide the details to integrate this endpoint in OAuth 2.0 implementations [[RFC6749](https://peppelinux.github.io/draft-demarco-nonce-endpoint/draft-demarco-nonce-endpoint.html#RFC6749)], our approach and the identified security considerations about Nonce are general.”

I added the suggested abstract in the PR #19

File: draft-demarco-oauth-nonce-endpoint.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ informative:
   BCP195: BCP195
 
 --- abstract
-This document defines the Nonce Endpoint for OAuth 2.0 implementations [RFC6749]. It details how an Authorization Server generates and issues opaque Nonces and how a client can learn about this endpoint to obtain a Nonce generated an Authorization Server or another entity.
+This document defines the Nonce Endpoint for OAuth 2.0 implementations [RFC6749]. It details how a Server generates and issues opaque Nonces and how a client can learn about this endpoint to obtain the Nonce.
```
Comment (OR13) [inline draft-demarco-oauth-nonce-endpoint.md:44]:
```suggestion
This document defines a Nonce Endpoint and details how a Server generates and issues opaque Nonces and how a client can learn about this endpoint to obtain the Nonce.
```

Comment (OR13) [reply draft-demarco-oauth-nonce-endpoint.md:44]: if the goal is to make this endpoint normatively citable from things other than OAuth.

Comment (peppelinux) [reply draft-demarco-oauth-nonce-endpoint.md:44]: @giadas ILGTM

Comment (giadas) [reply draft-demarco-oauth-nonce-endpoint.md:44]: For me it is ok as suggested by Orie. Minor: I suggest to have both entities lowercases "Server" -> "server" (or both capitalize "client" -> "Client")

**Pull request merged**